packages/build/bin/utils.js

```diff
@@ -12,11 +12,12 @@ const spawn = require('cross-spawn');
 const debug = require('debug')('loopback:build');
 
 /**
- * Get the Node.js compilation target - es2017 or es2015
+ * Get the Node.js compilation target - es2017
```

Comment:
IMO, we should keep `es2015` support for `@loopback/build` as it can be used to build TypeScript projects independent of `LoopBack 4`.

Reply:
+1

Also as I mentioned above, I'd like us to add es2018 target soon (around April this year, when Node.js 10.0.0 is released).

 - V8 6.4 already supports all ES2018 features according to [this tweet](https://twitter.com/v8js/status/956976954074267648)
 - Node's `master` branch contains V8 6.4 AFAICT, see [deps/v8](https://github.com/nodejs/node/tree/936eea5099bb5ab31b76ef22fcf15baed0cdf5ab/deps/v8)

Reply:
(I do realize this contradicts the checklist in https://github.com/strongloop/loopback-next/pull/945, which I created back in October last year.)